upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template

```diff
@@ -253,8 +253,6 @@ spec:
           operator: Exists
         - effect: NoExecute
           operator: Exists
-        - key: CriticalAddonsOnly
```

Comment:
Why the removal of this toleration?

Also maybe we should add a comment at the top that links to where this manifest (loosely) came from: https://github.com/weaveworks/weave/blob/9f00f78d3b9d5a8a31fdd90ec691095028e2690a/prog/weave-kube/weave-daemonset-k8s-1.11.yaml

Reply:
Some confusion from me. Added back and also the header.
One other thing, had to move from the 1.8 manifest (that doesn't exist anymore) to the 1.19 one.